src/passes/pass.cpp

```diff
@@ -698,6 +698,10 @@ void PassRunner::handleAfterEffects(Pass* pass, Function* func) {
 int PassRunner::getPassDebug() {
   static const int passDebug =
     getenv("BINARYEN_PASS_DEBUG") ? atoi(getenv("BINARYEN_PASS_DEBUG")) : 0;
+  int testVar = 5;
+  if (passDebug > 0)
+    testVar = 3;
```

Comment:
This is to show how Travis CI errors out in case of clang-tidy errors, and will not be included in the final commit